src/man/sssd.conf.5.xml

```diff
@@ -2030,6 +2030,21 @@ p11_uri = library-description=OpenSC%20smartcard%20framework;slot-id=2
             configuration section, that is, in a section called
             <quote>[domain/<replaceable>NAME</replaceable>]</quote>
             <variablelist>
+                <varlistentry>
+                    <term>enabled</term>
+                    <listitem>
+                        <para>
+                            Override the enabled/disabled state of the
+                            <quote>domains</quote> list at <quote>[sssd]
+                            </quote>. Set it to <quote>true</quote> if you want
+                            to enable the domain. Set it to <quote>false
+                            </quote>
+                            if you want to disable the domain from the domains
+                            list.
+                        </para>
+                    </listitem>
```

Comment:
I'm not very fond of this description. How about:

Explicitly enable or disable the domain. If true, the domain is always enabled. If false, the domain is always disabled. If this option is not set, the domain is enabled only if it is listed in the domains option in the [sssd] section.

Reply:
Much better! Thank you! Done